test/unit/search/search_collection_test.cpp

```diff
@@ -172,6 +172,40 @@ TYPED_TEST(search_test, multiple_queries)
     EXPECT_RANGE_EQ(search(queries, this->index, cfg) | query_id, expected_query_ids);
 }
 
+TYPED_TEST(search_test, parallel_queries)
+{
+    std::vector<std::vector<seqan3::dna4>> const queries{100, {"ACGTACGTACGT"_dna4}};
```

Comment:
Please, make the size a constexpr variable and use it in the test, so that it only needs to be changed once.,